src/main/java/uk/gov/hmcts/reform/bulkscanprocessor/entity/PaymentRepository.java

```diff
@@ -1,9 +1,20 @@
 package uk.gov.hmcts.reform.bulkscanprocessor.entity;
 
 import org.springframework.data.jpa.repository.JpaRepository;
+import org.springframework.data.jpa.repository.Modifying;
+import org.springframework.data.jpa.repository.Query;
+import org.springframework.data.repository.query.Param;
 
+import java.util.List;
+import java.util.Optional;
 import java.util.UUID;
 
 public interface PaymentRepository extends JpaRepository<Payment, UUID> {
 
+    @Modifying(clearAutomatically = true)
+    @Query("UPDATE Payment p SET p.status = 'REQUESTED'"
```

Comment:
`REQUESTED` why ?

Reply:
Payment is requested but it is action-ed sometime later by the PAYHUB

Reply:
do we have 1 status?

Reply:
yes as of now. Are you expecting any more status?

Reply:
from our perspective payment is done successfully. we better rename?

Reply:
Changed to SUBMITTED.